### ISSUE TITLE
Indentation leaks to next line when inside object

### ISSUE THREAD

**Issue by Naatan** [Komodo/KomodoEdit#2180]
### Steps to Reproduce

Create a javascript file with the following contents

```
var foobar = {
    k1: {
        
    },
    k2: {
        
    },
    k3: {
        
    },
};
```

Select lines 2 through 4 and press TAB 3 times

### Expected results

Lines 2 through 4 are indented 3 times.

### Actual results

Each time TAB is pressed it expands the selection by one line to the bottom, you'll end up with lines 2 through 7 selected.

**Comment by cavac** [comment]
Reproduced the Bug on Komodo IDE, version 10.1.2, build 89508, platform linux-x86_64.

**Comment by cgchoffman** [comment]
Nightly build with a fix for this will be available around 12 PST today.  I will update this ticket when it's available.

**Comment by cgchoffman** [comment]
IDE: http://downloads.activestate.com/Komodo/nightly/komodoide/latest-10.1.3/
Edit: http://downloads.activestate.com/Komodo/nightly/komodoedit/latest-10.1.3/

**Comment by cgchoffman** [comment]
FYI, once we release 10.1.3, if you have installed this nightly, Komodo will automatically prompt you to auto-update to the mainline release.

**Comment by cavac** [comment]
Confirmed: Nightly fixed the problem on Ubuntu 16.04 x64 for IDE.
